project/SparkBuild.scala

```diff
@@ -340,7 +340,7 @@ object SparkBuild extends PomBuild {
   val mimaProjects = allProjects.filterNot { x =>
     Seq(
       spark, hive, hiveThriftServer, hiveCompatibility, catalyst, repl, networkCommon, networkShuffle, networkYarn,
-      unsafe, testTags, sketch, mllibLocal
+      unsafe, testTags, sketch, mllibLocal, streamingKafka
```

Comment:
You don't necessarily have to disable MiMa here. Instead, you can override `MimaKeys.previousArtifact` in the `streamingKafka` project.